engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/compiler/FlinkProcessCompiler.scala

```diff
@@ -46,7 +46,9 @@ class FlinkProcessCompiler(creator: ProcessConfigCreator,
   def compileProcess(process: EspProcess,
                      processVersion: ProcessVersion,
                      deploymentData: DeploymentData,
-                     resultCollector: ResultCollector)(userCodeClassLoader: ClassLoader): FlinkProcessCompilerData = {
+                     resultCollector: ResultCollector,
+                     runMode: RunMode)
```

Comment:
It is sad that we need to pass this runMode during compilation :(. It should be done only in interpreter but to make it happen we should do a lot rework in definition / implementation split and proposed approach is the best for now.